examples/platforms/efr32/rail_config.c

```diff
@@ -0,0 +1,184 @@
+// Copyright 2018 Silicon Laboratories, Inc.
+//
```

Comment:
Not sure if the license is ok to be in this folder.

Reply:
I'm not sure too. This file is auto-generated by Hardware Configurator in Simplicity Studio.  @jwhui Can you provide some suggestions.

Reply:
Files that do not carry the OpenThread license header must be placed in `third_party`.

Reply:
Updated.